### PR TITLE
intercept $http respnoseError, show popup alerts stated in errror.json

### PR DESCRIPTION
**Pull request by shen-xujing**

### CODE REVIEW

File: beeline/services/ErrorInterceptor.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+angular.module("beeline").factory("ErrorInterceptor", [
+  "$injector",
+  "$q",
+  function($injector, $q) {
+    return {
+      // optional method
+      responseError: function(rejection) {
+        // do something on error
+        if (rejection.status >= 500) {
```
Comment (joelchoo) [inline beeline/services/ErrorInterceptor.js]: should we do `600 > rejection.status >= 500` instead?

Comment (shen-xujing) [reply beeline/services/ErrorInterceptor.js]: 6xx—Global Failure Responses, i don't think we need to exclude this

Comment (joelchoo) [reply beeline/services/ErrorInterceptor.js]: Okay sure!

**Comment by joelchoo** [comment]
Merge delayed until we really need this

**Comment by joelchoo** [comment]
Labelled with shelved for future reference so we're closing this issue

**Pull request closed without merging**